src/runtime/VMInstance.cpp

```diff
@@ -143,13 +143,13 @@ void VMInstance::gcEventCallback(GC_EventType t, void* data)
     const bool debuggerEnabled = false;
 #endif /* ESCARGOT_DEBUGGER */
 
-    if (t == GC_EventType::GC_EVENT_MARK_START && !debuggerEnabled) {
-        if (self->m_regexpCache->size() > REGEXP_CACHE_SIZE_MAX) {
+    if (t == GC_EventType::GC_EVENT_MARK_START && UNLIKELY(!debuggerEnabled)) {
```

Comment:
It seems that `debuggerEnabled` is set for debugger mode, so this is one of rare cases.
I think that `UNLIKELY` needs to be fixed as `LIKELY`.

Reply:
Oh!

Reply:
I updated it